build.sbt

```diff
@@ -218,11 +218,28 @@ lazy val `quill-cassandra-monix` =
     .settings(commonSettings: _*)
     .settings(mimaSettings: _*)
     .settings(
-      fork in Test := true,
+      fork in Test := true
     )
     .dependsOn(`quill-cassandra` % "compile->compile;test->test")
     .dependsOn(`quill-monix` % "compile->compile;test->test")
 
+lazy val `quill-cassandra-lagom` =
+   (project in file("quill-cassandra-lagom"))
+    .settings(commonSettings: _*)
+    .settings(mimaSettings: _*)
+    .settings(
+      fork in Test := true,
+      libraryDependencies ++= {
+        val lagomVersion = "1.5.0-RC1"
+        Seq(
+          "com.lightbend.lagom" %% "lagom-scaladsl-persistence-cassandra" % lagomVersion % Provided,
+          "com.lightbend.lagom" %% "lagom-scaladsl-testkit" % lagomVersion % Test
+        )
+      }
+    )
+    .dependsOn(`quill-cassandra` % "compile->compile;test->test")
```

Comment:
Does the lagom driver depend on the datastax driver? If not we shouldn't have this module depend transitively on the datastax driver

Reply:
cool! LGTM